manager/orchestrator/constraintenforcer/constraint_enforcer.go

```diff
@@ -83,10 +84,15 @@ func (ce *ConstraintEnforcer) rejectNoncompliantTasks(node *api.Node) {
 		log.L.WithError(err).Errorf("failed to list tasks for node ID %s", node.ID)
 	}
 
-	var availableMemoryBytes, availableNanoCPUs int64
+	available := &api.Resources{}
+	fakeStore := make([]*api.GenericResource, 0)
```

Comment:
The fakestore is the replacement for nodeAvailable. We don't need to maintain a list of available resource here so I'm creating a "fakestore: